r120/u_inref/ydb_dbglvl.csh

```diff
@@ -0,0 +1,33 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+echo "# Test for ydb_dbglvl env var"
+
+#
+echo '# Test that gtmdbglvl=2 (print usage statistics at process end) controls ydbDebugLevel if ydb_dbglvl is undefined'
+unsetenv ydb_dbglvl
+setenv gtmdbglvl 2
+$gtm_exe/mumps -run %XCMD 'write "ydbDebugLevel=",ydbDebugLevel,!' |& $grep "small storage performance"
```

Comment:
Why use a code fragment that generates an error here (ydbDebugLevel is not defined) instead of just doing say "set a=1+2" for example.

Reply:
I did not even realize the error until you mentioned. But thinking about it, since I am grepping for a specific string whose output is dependent only on ydb_dbglvl setting and not on the actual M line executed, I hope you are okay with any M line. Let me know if not and I will fix it.